e2e_tests/tests/experiment/test_custom_searcher.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from determined.common.api import bindings
+from tests import config as conf
+from tests import experiment as exp
+
+
+@pytest.mark.e2e_cpu
+def test_get_searcher_ops() -> None:
+    exp_id = exp.run_basic_test(
+        conf.fixtures_path("no_op/single.yaml"), conf.fixtures_path("no_op"), 1
+    )
+    sess = exp.determined_test_session()
+    bindings.get_GetSearcherEvents(sess, experimentId=exp_id)
+
+
+@pytest.mark.e2e_cpu
+def test_post_searcher_ops() -> None:
+    exp_id = exp.run_basic_test(
+        conf.fixtures_path("no_op/single.yaml"), conf.fixtures_path("no_op"), 1
+    )
+    sess = exp.determined_test_session()
+    op1 = bindings.v1SearcherOperation(
+        createTrial="createTrial"
+    )  # what do we need to include for an operation such as createTrial?
```

Comment:
I would assume you need to include everything to instantiate a master-side operation: https://github.com/determined-ai/determined/blob/97d3e6029eed98cc4ab073097753df5eff32a693/master/pkg/searcher/operations.go#L106

Reply:
I see that Liam answered your question: only Hparams need to be communicated by the client; the rest is available on master.

Reply:
Yes making that change! Thanks Maksim.